Jenkinsfile

```diff
@@ -738,7 +738,9 @@ def rsyncResults(stage) {
     }
 }
 
-def checkLastImage() {
+def checkLastImage(stage) {
```

Comment:
I just took what was there before that touched a file.  In the shell step how do you return that to the method being called.